### PR TITLE
Document edgedb instance credentials (#6685)

### PR DESCRIPTION
**Pull request by MiroslavPetrik**

### CODE REVIEW

**Review by Dhghomon — APPROVED** [review]
Thanks for the PR! Matches the wording and tone of the other pages entirely.

**Review by Dhghomon — APPROVED** [review]

File: docs/cli/edgedb_instance/edgedb_instance_credentials.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.. _ref_cli_edgedb_instance_credentials:
+
+
+===========================
+edgedb instance credentials
+===========================
+
+Display instance credentials (add `--json` for verbose).
```
Comment (Dhghomon) [inline docs/cli/edgedb_instance/edgedb_instance_credentials.rst:8]: Oh right, also double backticks here as the test mentions

Comment (MiroslavPetrik) [reply docs/cli/edgedb_instance/edgedb_instance_credentials.rst]: Fixed https://github.com/edgedb/edgedb/pull/6686/commits/8a2fb7a5ea5ba929692243632968f3349fa6af53

**Review by Dhghomon — APPROVED** [review]

File: docs/cli/edgedb_instance/edgedb_instance_credentials.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.. _ref_cli_edgedb_instance_credentials:
+
+
+===========================
+edgedb instance credentials
+===========================
+
+Display instance credentials (add ``--json`` for verbose).
```
Comment (raddevon) [inline docs/cli/edgedb_instance/edgedb_instance_credentials.rst:8]: I think we should move the note about the `--json` option to the "Options" section.
```suggestion
Display instance credentials.
```

(I know it's presented as you have it here in the CLI's `--help` output, so I'm not faulting you for it in any way.)

File: docs/cli/edgedb_instance/edgedb_instance_credentials.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+``edgedb instance credentials`` is a terminal command for displaying the
+credentials of an EdgeDB instance.
+
+
+Options
+=======
+
+:cli:synopsis:`--json`
+    Output in JSON format (password is included in cleartext).
```
Comment (raddevon) [inline docs/cli/edgedb_instance/edgedb_instance_credentials.rst:26]: This completes the change suggested above and adds note of the TLS certs being included in the output here.

```suggestion
    Output in JSON format. In addition to formatting the credentials as JSON,
    this option also includes the password in cleartext and the TLS
    certificates.
```

**Review by raddevon — CHANGES_REQUESTED** [review]
Great work here! I suggested two small changes. If you don't mind to take a look at those, I can get it merged for you.

**Comment by MiroslavPetrik** [comment]
@raddevon ping

**Review by raddevon — APPROVED** [review]
Thank you for contributing this!

**Pull request merged**